models/model_choice.py

```diff
@@ -57,10 +57,18 @@ def net(net_params, num_channels, inference=False):
             model = models.segmentation.deeplabv3_resnet101(pretrained=False, progress=True, in_channels=num_bands,
                                                             num_classes=num_channels, aux_loss=None)
         except:
-            assert num_bands==3, 'Edit torchvision scripts segmentation.py and resnet.py to build deeplabv3_resnet ' \
-                                 'with more or less than 3 bands'
-            model = models.segmentation.deeplabv3_resnet101(pretrained=False, progress=True,
-                                                            num_classes=num_channels, aux_loss=None)
+            # assert num_bands==3, 'Edit torchvision scripts segmentation.py and resnet.py to build deeplabv3_resnet ' \
+            #                      'with more or less than 3 bands'
+            print('Testing with 4 bands')
+            import numpy as np
+            import torch.nn as nn
```

Comment:
Add those imports at the beginning of the script.

Reply:
Would resolve